### ISSUE TITLE
Hangs on Macrumors feed

### ISSUE THREAD

**Issue by macuser1234** [ViennaRSS/vienna-rss#1243]
Using Vienna Version 3.5.4 :8b11d92d: (7074) on Mac OS X 10.12.6

Have been having trouble for some time with the MacRumors feed:
http://feeds.macrumors.com/MacRumors-Front

Certain articles when they come into view will cause Vienna to to hang up with the rainbow spinning pinwheel and require a force quit. For example:

Alleged Leaked CAD Image Suggests 'iPad mini 5' Will Just Be a Specs Bump (2/19/19)
Apple Pay Rolling Out in Czech Republic and Saudi Arabia (2/19/19)

**Comment by barijaona** [comment]
I could not yet reproduce the problem.

Is the trouble happening while you read the article list in the main tab or when you open the specific article on a separate tab ?
If it is the former case, which style do you use ? Is 'Use Web Page for Articles' enabled ?

**Comment by macuser1234** [comment]
The problem is when reading in the article list in the main tab.
I did a little more testing, and I isolated one of the trouble articles as "Apple Pay Rolling Out in Czech Republic and Saudi Arabia (2/19/19)"

1) Subscribe to feed http://feeds.macrumors.com/MacRumors-Front
2) Filter by "All Articles"
3) Search for article "Apple Pay Rolling"
4) The above article should be the top hit, and causes a freeze with a spinning rainbow.

However, you are correct, it does not happen in every style (to answer your other question,  I have Use Web Page for Articles" disabled, but this does not seem to make a difference).

Examples of styles that do not cause a freeze: Classy, Serafim Dark
Examples of styles that do cause a freeze: Default, Feedlight Aqua (the one I was originally using), Minimilista

I am happy to change my style to avoid this freeze, but would be curious why this articlemight cause some particular styles to freeze.
